### ISSUE TITLE
Feature: Support URL paths when logging in

### ISSUE THREAD

**Issue by jackjamieson2** [gRegorLove/indiebookclub#8]
When I try to log in with a path in my URL indiebookclub returns an error:

> Invalid “me” parameter
> 
> URL paths are not currently supported. Please log in with only a domain name.

Being able to log in with a path would be helpful for troubleshooting because (1) my testing site uses a path in the URL and (2) [micropub.rocks](https://micropub.rocks) creates a temporary login with a path for testing clients.  e.g. https://micropub.rocks/client/[key]

Background: I was trying to troubleshoot a problem I'm having with indiebookclub posts in wordpress. That problem surfaces in two ways (1) Wordpress's post editor seems to only partially load for posts created in indiebookclub (as I briefly mentioned in #6), (2) Post titles are prefaced with the text 'array'.  This problem goes away when I disable the post kinds plugin, so I think it's probably something there but I wanted to investigate a bit further in case it's an issue with indiebookclub.

**Comment by gRegorLove** [comment]
This is something I can't get around easily currently because the `me` is used to create the profile slug, e.g. https://indiebookclub.biz/users/gregorlove.com

**Comment by jackjamieson2** [comment]
Ah, that makes sense. Low priority anyway.  I just thought I’d ask in case
it was a simple change. Thanks!

**Comment by gRegorLove** [comment]
Yeah, sorry about that. I mentioned the issues you're having in the WordPress chat, though, so hopefully we can figure out the array-to-string conversion issue.

**Comment by gRegorLove** [comment]
I'm keeping this open to think through possible solutions:

* replace slashes in URL path with hyphens. easy to do, but could make long profile URLs
* create a short hash of the path and make the profile slug `example.com-hash`
* ...

**Comment by jackjamieson2** [comment]
To add more brainstorming:

- The profile slug could be an editable username field. It could default to the URL path (using hyphens instead of slashes) but provide the option to change it to something shorter if desired.  (This would increase complexity, so may not be suitable)

**Comment by snarfed** [comment]
👬 snarfed/bridgy#629 snarfed/bridgy#652

**Comment by djwudi** [comment]
Just adding a "+1" or "me too!" or whatever is appropriate for this. I've been looking at Goodreads alternatives, started looking at Indiebookclub, but then ran into this login error, as my blog is under a directory on my domain (michaelhans.com/eclecticism).

**Comment by gRegorLove** [comment]
Another +1 for https://zylstra.org/blog
